backend/app.py

```
@@ -1101,6 +1122,34 @@ async def get_match(request: Request, match_id: int):
         )
 
 
+@app.delete("/api/matches/{match_id}")
+async def delete_match(
+    request: Request,
+    match_id: int,
+    current_user: dict[str, Any] = Depends(require_admin)
```

Ruff (0.14.1)

Unused function argument: `current_user`

(ARG001)